wiki/en/Getting-Started.md

```diff
@@ -62,7 +66,7 @@ Now use the “Connect” button in the Jamulus main window to join a server. A
 
 The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 15ms if you can.
```

Comment:
```suggestion
The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 50ms if you can.
```

Increasing the reccommended ping time might encourage people to use more public servers, as @jujudusud points out.

Reply:
50ms is quite high… 

30ms? However, it all depends on the hardware too.

Reply:
As discussed with pljones, 50ms is the real maximum we had used for a server.
We want to encourage users to connect to the existing public server.

Reply:
Ok. So applied that and merged PR.